NotesApp/Scenes/Editor/EditorViewController.swift

```diff
@@ -80,42 +89,31 @@ class EditorViewController: UIViewController {
     private func setupUI() {
         view.backgroundColor = UIColor(hex: "#252525")
         
-        setupCustomBackButton()
-        
         titleTextView.addDoneButtonToolbar()
         bodyTextView.addDoneButtonToolbar(onDone: (target: self, action: #selector(customDoneAction)))
         
         view.addSubview(titleTextView)
```

Comment:
You could make extension for Uiview like this: extension UIView {
    func addSubviews(_ subviews: UIView...) {
        subviews.forEach { subview in
            addSubview(subview)
        }
    }
} and it will be better to have method setupViewHierarchy or something similar where you will add components to view and then call it in setupUi()